.solhint.json

```diff
@@ -5,7 +5,7 @@
     "compiler-version": ["error", ">=0.8.25"],
     "func-name-mixedcase": "off",
     "func-visibility": ["error", { "ignoreConstructors": true }],
-    "max-line-length": ["error", 120],
+    "max-line-length": "off",
```

Comment:
Is ok for everyone to keep only the fmt max-line-length check?

Reply:
yes